app/views/data_exports/show.html.erb

```diff
@@ -44,34 +44,46 @@
 <% end %>
 
 <div class="bg-white dark:bg-slate-800">
-  <div class="flex flex-row">
-    <%= render Pathogen::TabsPanel.new(
-      id: "data-export-tabs",
-      label: t(:"data_exports.show.tabs.title")
-    ) do |tabs| %>
+  <%= render Pathogen::Tabs.new(
+    id: "data-export-tabs",
+    label: t(:"data_exports.show.tabs.title"),
+    default_index: @tab == "preview" ? 1 : 0,
```

Comment:
Suggestion (non-blocking): should we set this in the controller instead like done in other places?